desc/io/input_reader.py

```diff
@@ -834,7 +834,32 @@ def parse_vmec_inputs(vmec_fname, threshold=0):  # noqa: C901 - FIXME: simplify
         pres_scale = 1.0
         curr_tor = None
 
-        for line in vmec_file:
+        # Loop which makes multi-line inputs a single line
+        vmeclines_no_multiline = []
+
+        for line in vmec_file.readlines():
+            comment = line.find("!")
+            line = (line.strip() + " ")[0:comment].strip()
```

Comment:
don't we want to completely ignore a line if it starts with `!`?

Reply:
this line does that if `!` is at the begining, and if it is later in the line it includes everything up until that point in the line then ignores the rest